LibreHardwareMonitorLib/Hardware/Gpu/IntelGpuGroup.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace LibreHardwareMonitor.Hardware.Gpu
+{
+    internal class IntelGpuGroup : IGroup
+    {
+        private readonly List<Hardware> _hardware = new();
+        private readonly StringBuilder _report = new();
+
+        public IntelGpuGroup(CPU.IntelCpu intelCpu, ISettings settings)
+        {
+            if (!Software.OperatingSystem.IsUnix)
+            {
+                _report.AppendLine("Intel GPU (D3D)");
+                _report.AppendLine();
+
+                var ids = D3DDisplayDevice.GetDeviceIdentifiers();
+                
+                _report.Append("Number of adapters: ");
+                _report.AppendLine(ids.Length.ToString(CultureInfo.InvariantCulture));
+                _report.AppendLine();
+
+                for (var i = 0; i < ids.Length; i++)
+                {
+                    var deviceId = ids[i];
+                    var isIntel = deviceId.IndexOf("VEN_8086") != -1;
+
+                    _report.Append("AdapterIndex: ");
+                    _report.AppendLine(i.ToString(CultureInfo.InvariantCulture));
+                    _report.Append("DeviceId: ");
+                    _report.AppendLine(deviceId);
+                    _report.Append("IsIntel: ");
+                    _report.AppendLine(isIntel.ToString(CultureInfo.InvariantCulture));
+
+                    if (isIntel)
+                    {
+                        if (D3DDisplayDevice.GetDeviceInfoByIdentifier(deviceId, out D3DDisplayDevice.D3DDeviceInfo deviceInfo, "GPU "))
```

Comment:
This was kept at D3D for a reason as it's the D3D information (same as for the other GPUs).

Reply:
Yeah, the usage of prefixes in sensor names seems very inconsistent to me, I thought this made more sense, but can use D3D

Reply:
These are all D3D information, it doesn't necessarily represent actual GPU values and would be confusing. For example, some of the memory used values can be different in GPU memory due to compression.